terraform/a2ha-terraform/modules/habitat/templates/install_hab.sh.tpl

```diff
@@ -129,7 +129,7 @@ tar xf ${aib_file} -C ${tmp_path}/aib_workspace || failure "Failed to extract ${
 rsync -a --keep-dirlinks ${tmp_path}/aib_workspace/hab /
 
 # Copy the hab bin if it doesn't exist
-[ ! -f /usr/bin/hab ] && rsync -a ${tmp_path}/aib_workspace/bin/hab /usr/bin
+rsync -a ${tmp_path}/aib_workspace/bin/hab /usr/bin
```

Comment:
I'm thinking this should probably be a symlink instead, we've run into this before where if the hab executable is an file instead of a symlink then when a new hab package version is installed it won't get updated when using the `--binlink` install option.